sys/net/sock/sock_util.c

```diff
@@ -117,19 +122,24 @@ int sock_urlsplit(const char *url, char *hostport, char *urlpath)
     }
 
     char *pathstart = _find_pathstart(hoststart);
-    if(!pathstart) {
-        return -EINVAL;
-    }
 
-    memcpy(hostport, hoststart, pathstart - hoststart);
+    size_t hostlen = pathstart - hoststart;
+    /* hostlen must be smaller SOCK_HOSTPORT_MAXLEN to have space for the null
+     * terminator */
+    if (hostlen >= SOCK_HOSTPORT_MAXLEN) {
```

Comment:
I first missed the `>=` but was correctly justified by the comment so thought it was only not my style.

But I noticed it is not consistent with the change below, so would prefer to use the same style, either both `>=` or `> MAXLEN -1`.

Reply:
Done